modules/bmo-about-the-baremetalhost-resource.adoc

```diff
@@ -94,7 +94,7 @@ a|  (Optional) Contains the information about the RAID configuration for bare me
 
 [NOTE]
 ====
-{product-title} {product-version} supports hardware RAID for BMCs using the iRMC protocol only. {product-title} {product-version} does not support software RAID.
+{product-title} {product-version} supports hardware RAID for BMCs using the iRMC protocol with support for RAID levels 0, 1, 5, 6, and 10, and the Redfish API with the iDRAC protocol with firmware version 6.10.30.20 or later and RAID levels 0, 1, and 5. {product-title} {product-version} does not support software RAID.
```

Comment:
It should be: "Hardware RAID configuration for RAID levels 0, 1 and 5 is also supported for the iDRAC BMC (with firmware version 6.10.30.20 or later), using the Redfish protocol." :  iDRAC is a BMC implementation and we are using the Redfish protocol to configure the RAID. There is no iDRAC protocol.